extensions/vscode/.eslintrc.json

```diff
@@ -26,7 +26,8 @@
     "curly": "warn",
     "eqeqeq": "warn",
     "no-throw-literal": "warn",
-    "semi": "off"
+    "semi": "off",
+    "require-await": "error"
```

Comment:
I really like how this cleanup included a rule change to our linting to ensure it was followed. ❤️